context("geom-ref-line")

test_that("can generate error messages", {
  expect_error(geom_ref_line(), "one of")
})
